core/src/App.html

```diff
@@ -209,13 +160,14 @@
   let resizeTabNavToggle = false;
   let thirdPartyCookiesCheck;
   let searchProvider;
-  let userSettingsInternalObj;
+  let internalUserSettingsObject;
```

Comment:
```suggestion
  let internalUserSettingsObject = {};
```
Combine [line 968](https://github.com/SAP/luigi/pull/1802/files#diff-c7697a12c9830e85a6f12e0bf2c589866d6d878a5f6f2c594fd4540937b09229R968) together.

Reply:
👍